azurerm_application_insights_web_test/main.tf

```diff
@@ -0,0 +1,34 @@
+# The module configure an web test from an url
+
+# Existing infrastructure
+data "azurerm_resource_group" "rg" {
+  name = "${local.azurerm_resource_group_name}"
+}
+
+# New infrastructure
+resource "azurerm_application_insights_web_test" "web_test" {
+  count = "${length(var.web_tests)}"
+
+  name = "${lookup(var.web_tests[count.index], "name")}"
+
+  resource_group_name     = "${local.azurerm_resource_group_name}"
+  application_insights_id = "/subscriptions/ec285037-c673-4f58-b594-d7c480da4e8b/resourceGroups/io-dev-rg/providers/microsoft.insights/components/io-dev-ai-generic-web-tests"
```

Comment:
Can we use the the application_insights data source to retrieve this information? https://www.terraform.io/docs/providers/azurerm/d/application_insights.html